rules-reviewed/jboss-eap/uncategorized/tests/data/MyClass.java

```diff
@@ -12,7 +14,10 @@ public MyClass(){
    public void myMethod() {
 	Class.forName("environment.dependent.MyClass");
         Class.forName("environment.dependent.OtherClass");
-        InitialContext context = new InitialContext();
+        Properties environment = new Properties();
```

Comment:
I'd like to note that I am testing this with Properties that is a subclass of Hashtable and it works like a charm (I was not sure 100 percent sure if we support this).